README.md

```diff
@@ -81,6 +81,42 @@ Inspect the pods within the network using:
 
 >HINT: Use tools like [kubectx](https://github.com/ahmetb/kubectx) or [kubie](https://github.com/sbstp/kubie) to work with multiple namespaces and contexts.
 
+### Specifying a Ceramic admin secret
+
+You can choose to specify a private key for the Ceramic nodes to use as their admin secret. This will allow you to set
+up the corresponding DID with CAS Auth.
+
+Leaving the private key unspecified will cause a new key to be randomly generated. This can be fine for simulation runs
+against CAS/Ganache running locally within the cluster but not for simulations that hit CAS running behind the AWS API
+Gateway. Using an unauthorized DID in that case will prevent the Ceramic nodes from starting up.
+
+```yaml
+apiVersion: "keramik.3box.io/v1alpha1"
+kind: Network
+metadata:
+  name: small
+spec:
+  replicas: 2
+  ceramic:
+    privateKeySecret: "private-key"
```

Comment:
should this be `small` as that is the name of the secret?

Reply:
Done